index.html

```diff
@@ -115,6 +117,93 @@ <h4 class="heading heading_level-4">Подушка</h4>
+                        <li class="column column_s-2 column_m-2 column_l-4">
+                            <div class="services__item">
+                                <img src="./styles/blocks/services/delivery.svg" class="services__item-ico" alt="Быстрая доставка по всей России">
+                                Быстрая доставка по всей России
+                            </div>
+                        </li>
+                        <li class="column column_s-2 column_m-2 column_l-4">
+                            <div class="services__item">
+                                <img src="./styles/blocks/services/return.svg" class="services__item-ico" alt="100% гарантия возврата и подлинность товара">
+                                100% гарантия возврата и подлинность товара
+                            </div>
+                        </li>
+                        <li class="column column_s-2 column_m-2 column_l-4">
+                            <div class="services__item">
+                                <img src="./styles/blocks/services/support.svg" class="services__item-ico" alt="100% гарантия возврата и подлинность товара">
+                                Круглосуточная служба поддержки
+                            </div>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+        <footer class="footer">
+            <div class="column-wrapper">
+                <div class="column-row">
+                    <div class="footer__wrapper">
+                        <div class="column column_s-2 column_m-2 column_l-4">
+                            <ul class="social">
+                                <li class="social__item">
+                                    <a class="social__icon social__icon_type_facebook" href="#"></a>
+                                </li>
+                                <li class="social__item">
+                                    <a class="social__icon social__icon_type_twitter" href="#"></a>
+                                </li>
+                                <li class="social__item">
+                                    <a class="social__icon social__icon_type_vk" href="#"></a>
+                                </li>
+                                <li class="social__item">
+                                    <a class="social__icon social__icon_type_instagram" href="#"></a>
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="column column_s-2 column_m-4 column_l-4">
+                            <ul class="stores">
+                                <li class="stores__item column column_s-1 column_m-1">
+                                    <a href="#" target="_blank" class="stores__icon stores__icon_type_appstore "></a>
+                                </li>
+                                <li class="stores__item column column_s-1 column_m-1">
```

Comment:
эти классы тут не нужны ```column column_s-1 column_m-1``` 
```suggestion
                                <li class="stores__item">
```